src/_reducers/HelloWorldReducer.js

```diff
@@ -0,0 +1,15 @@
+import StateStorage from '../_store/StateStorage';
```

Comment:
This is a custom built thing by me, we should be replacing it with redux-localstorage (which seems to be better).